config/initializers/restricted_access.rb

```diff
@@ -0,0 +1,8 @@
+class HasRestrictedAccessConstraint
+  def self.matches?(request)
+    return true if Rails.env.development?
+
+    current_user = request.env['warden'].user
```

Comment:
I don't know if door keeper uses warden. I did not set up Devise on this app cuz I don't think there's a reason for it. My thought is we are trying to simplify login as much as possible and this includes introducing something like devise unless we needed to